src/univariate/continuous/generalizedpareto.jl

```diff
@@ -15,8 +15,9 @@ f(x; \\mu, \\sigma, \\xi) = \\begin{cases}
 ```
 
 ```julia
-GeneralizedPareto()             # Generalized Pareto distribution with unit shape and unit scale, i.e. GeneralizedPareto(0, 1, 1)
-GeneralizedPareto(μ, σ)         # Generalized Pareto distribution with shape ξ and scale σ, i.e. GeneralizedPareto(0, σ, ξ)
+GeneralizedPareto()             # Standard exponential distribution (GPD with shape 0, location 0, scale 1)
```

Comment:
This is wrong. You can just change it back to the current version.
```suggestion
GeneralizedPareto()             # Generalized Pareto distribution with unit shape and unit scale, i.e. GeneralizedPareto(0, 1, 1)
```